python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -592,13 +592,21 @@ class ScarletDeblendConfig(pexConfig.Config):
 
     # Size restrictions
     maxNumberOfPeaks = pexConfig.Field(
-        dtype=int, default=0,
+        dtype=int, default=200,
         doc=("Only deblend the brightest maxNumberOfPeaks peaks in the parent"
              " (<= 0: unlimited)"))
     maxFootprintArea = pexConfig.Field(
-        dtype=int, default=1000000,
+        dtype=int, default=int(1e5),
         doc=("Maximum area for footprints before they are ignored as large; "
              "non-positive means no threshold applied"))
+    maxAreaTimesPeaks = pexConfig.Field(
+        dtype=int, default=int(1e7),
```

Comment:
Similarly, this can be `10_000_000`.  Also, I think the default should be on a new line.